extensions/monitor/__init__.py

```diff
@@ -36,7 +34,14 @@ def __init__(self, bot):
 
     async def on_member_update(self, before, after):
         if before.display_name != after.display_name:
-            await self.post_to_bot_events(after, "<@{0.id}> {0.display_name} is now known as {1.display_name}".format(before, after))
+            channel = discord.utils.find(lambda c: c.name == "botevents", after.guild.channels)
+            embed = discord.Embed(
```

Comment:
this should have `if channel != null:` before it

right now if it can't find #botevents it will crash